packages/botonic-plugin-contentful/src/nlp/stemmers/polish-stemmer.ts

```diff
@@ -0,0 +1,187 @@
+import { BaseStemmer } from 'node-nlp/lib/nlp/stemmers'
+import { Tokenizer } from 'node-nlp/lib/nlp/tokenizers'
+
+/**
+ * From https://github.com/Tutanchamon/pl_stemmer/blob/master/pl_stemmer.py
+ */
+export class PolishStemmer implements BaseStemmer {
+  constructor(private readonly tokenizer: Tokenizer) {}
+
+  tokenizeAndStem(str: string, keepStops: boolean): string[] {
+    const tokens = this.tokenizer.tokenize(str, false)
```

Comment:
what is doing this flag in this case? can we have this documented? I have seen same usages in methods `tokenizeAndStem` and in class `NormalizedUtterance`, maybe would we nice to know why is it used in this way in these functions/methods.